genkeys.go

```diff
@@ -70,6 +72,8 @@ func init() {
 		"RightBracket": {"BracketRight"},
 		"GraveAccent":  {"Backquote"},
 		"NumLock":      {"NumLock"},
+		"ScrollLock":   {"ScrollLock"},
+		"Menu":         {"Menu"},
```

Comment:
`"Menu": {"ContextMenu"}` (GLFW key is the key, KeyboardEvent.code is the value)